From: "David S. Miller" <davem@davemloft.net>
Date: Tue, 26 Jul 2016 23:19:29 -0700
Subject: xgene: Fix build warning with ACPI disabled.
Git-commit: 36232012344b8db67052432742deaf17f82e70e6
Patch-mainline: v4.8-rc1
References: bsc#993051

drivers/net/ethernet/apm/xgene/xgene_enet_hw.c: In function 'xgene_enet_phy_connect':
drivers/net/ethernet/apm/xgene/xgene_enet_hw.c:759:22: warning: unused variable 'adev' [-Wunused-variable]

Fixes: 8089a96f601b ("drivers: net: xgene: Add backward compatibility")
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 drivers/net/ethernet/apm/xgene/xgene_enet_hw.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/apm/xgene/xgene_enet_hw.c b/drivers/net/ethernet/apm/xgene/xgene_enet_hw.c
index 8a2a221..7714b7d 100644
--- a/drivers/net/ethernet/apm/xgene/xgene_enet_hw.c
+++ b/drivers/net/ethernet/apm/xgene/xgene_enet_hw.c
@@ -756,7 +756,6 @@ int xgene_enet_phy_connect(struct net_device *ndev)
 	struct device_node *np;
 	struct phy_device *phy_dev;
 	struct device *dev = &pdata->pdev->dev;
-	struct acpi_device *adev;
 	int i;
 
 	if (dev->of_node) {
@@ -781,7 +780,7 @@ int xgene_enet_phy_connect(struct net_device *ndev)
 		pdata->phy_dev = phy_dev;
 	} else {
 #ifdef CONFIG_ACPI
-		adev = acpi_phy_find_device(dev);
+		struct acpi_device *adev = acpi_phy_find_device(dev);
 		if (adev)
 			pdata->phy_dev =  adev->driver_data;
 
-- 
2.6.6

